### ISSUE TITLE
Bad signing key for Ubuntu Bionic repo

### ISSUE THREAD

**Issue by conorbdaly** [phusion/passenger#2142]
When trying to run apt-get update/upgrade before installing passenger on ubuntu/bionic, we get the following error:

Reading package lists... Done  
W: GPG error: https://oss-binaries.phusionpassenger.com/apt/passenger bionic Release: The following signatures were invalid: BADSIG 561F9B9CAC40B2F7 Phusion Automated Software Signing (Used by automated tools to sign software packages) <auto-software-signing@phusion.nl>
E: The repository 'https://oss-binaries.phusionpassenger.com/apt/passenger bionic Release' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

**Comment by vcsjones** [comment]
Same.

```
GPG error: https://oss-binaries.phusionpassenger.com/apt/passenger bionic Release: The following signatures were invalid: BADSIG 561F9B9CAC40B2F7 Phusion Automated Software Signing (Used byautomated tools to sign software packages) <auto-software-signing@phusion.nl>
E: The repository 'https://oss-binaries.phusionpassenger.com/apt/passenger bionic Release' is not signed
```

**Comment by gr8bit** [comment]
Anyone mailed that address yet?

**Comment by vcsjones** [comment]
Work around to unblock CI for now:

`rm /etc/apt/sources.list.d/passenger.list`

**Comment by FooBarWidget** [comment]
We were performing an update on the GPG key, looks like something went wrong. This has been fixed now.

**Comment by xcpdq** [comment]
Seems the issue is back:
on apt update:

Reading package lists... Done
W: GPG error: https://oss-binaries.phusionpassenger.com/apt/passenger bionic Release: The following signatures were invalid: BADSIG 561F9B9CAC40B2F7 Phusion Automated Software Signing (Used by automated tools to sign software packages) <auto-software-signing@phusion.nl>
E: The repository 'https://oss-binaries.phusionpassenger.com/apt/passenger bionic Release' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

**Comment by CamJN** [comment]
Thanks, looking into it.

**Comment by samcoinc** [comment]
is there a work around?

**Comment by CamJN** [comment]
Until I can figure out what went wrong, moving the passenger repo away from `/etc/apt/sources.list.d/passenger.list` to a backup location will unblock other apt usage. I don't have a workaround that will allow interacting with the Passenger repo yet.

**Comment by samcoinc** [comment]
Ok - I will wait.  this is a new install..

**Comment by samcoinc** [comment]
different error now.. Progress?  :)
Reading package lists... Done
W: Failed to fetch https://oss-binaries.phusionpassenger.com/apt/passenger/dists/bionic/InRelease  503  Service Unavailable [IP: 109.107.35.58 443]
W: Some index files failed to download. They have been ignored, or old ones used instead.

**Comment by samcoinc** [comment]
working here now..  Thanks!

**Comment by CamJN** [comment]
no problem, thanks to you and @xcpdq for the heads-up.

**Comment by samcoinc** [comment]
well - no.  It isn't again..  but it worked for a bit and I was able to get it installed. 

Reading package lists... Done
W: Failed to fetch https://oss-binaries.phusionpassenger.com/apt/passenger/dists/bionic/InRelease  503  Service Unavailable [IP: 109.107.35.58 443]
W: Some index files failed to download. They have been ignored, or old ones used instead.

**Comment by samcoinc** [comment]
omg - it is working now again..  I am signing off for the night.  Thanks again

**Comment by brettwgreen** [comment]
This issue still persists for us as of this morning 9am EST.

**Comment by CamJN** [comment]
I think we got this under control, please check on your end everybody.

**Comment by blqke** [comment]
Hello
I Still have the issue: 
```
$ sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 561F9B9CAC40B2F7                                              19:05:22

Executing: /tmp/apt-key-gpghome.xMfxZ36mLj/gpg.1.sh --keyserver keyserver.ubuntu.com --recv-keys 561F9B9CAC40B2F7
gpg: key 561F9B9CAC40B2F7: 2 signatures not checked due to missing keys
```

```
~/pitchme-am-app
$ sudo apt update                                                                                                             19:08:36
Hit:1 http://mirrors.digitalocean.com/ubuntu bionic InRelease
Get:2 http://mirrors.digitalocean.com/ubuntu bionic-updates InRelease [88.7 kB]
Hit:3 https://repos.sonar.digitalocean.com/apt main InRelease
Ign:4 https://oss-binaries.phusionpassenger.com/apt/passenger bionic InRelease
Get:5 http://mirrors.digitalocean.com/ubuntu bionic-backports InRelease [74.6 kB]
Get:6 https://oss-binaries.phusionpassenger.com/apt/passenger bionic Release [22.8 kB]
Get:7 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]
Err:8 https://oss-binaries.phusionpassenger.com/apt/passenger bionic Release.gpg
  503  Service Unavailable [IP: 109.107.35.58 443]
Reading package lists... Done
E: The repository 'https://oss-binaries.phusionpassenger.com/apt/passenger bionic Release' is not signed.
```

**Comment by balusarakesh** [comment]
I'm also seeing the same error just like @blqke

**Comment by balusarakesh** [comment]
thanks, it is working now

**Comment by inomire** [comment]
I got the same error just like above.

**Comment by inomire** [comment]
Now it works.

**Comment by vignesh1rajan** [comment]
Hi I am getting the following error with apt-update

The repository 'https://oss-binaries.phusionpassenger.com/apt/passenger bionic Release' does not have a Release file.

**Comment by CamJN** [comment]
@vignesh1rajan it was likely a cache issue, it seems to have resolved itself.

**Comment by ydorLorenzo** [comment]
Hi I'm getting this error with apt update

The repository 'https://oss-binaries.phusionpassenger.com/apt/passenger bionic Release' does not have a Release file.
Could anybody help me?

**Comment by soulraven** [comment]
same problem with Buster,
> Get:9 https://oss-binaries.phusionpassenger.com/apt/passenger buster Release [22.6 kB]
> Get:10 https://oss-binaries.phusionpassenger.com/apt/passenger buster Release.gpg
> Err:10 https://oss-binaries.phusionpassenger.com/apt/passenger buster Release.gpg
>   Connection timed out [IP: 109.107.35.58 443]
> Reading package lists... Done
> E: The repository 'https://oss-binaries.phusionpassenger.com/apt/passenger buster Release' is not signed.
> N: Updating from such a repository can't be done securely, and is therefore disabled by default.
> N: See apt-secure(8) manpage for repository creation and user configuration details.

**Comment by gr8bit** [comment]
Issues like this seem to happen quite regularly for different reasons. Today it seems the repo server isn't reachable.

**Comment by soulraven** [comment]
the ping and the web server is OK:

> ping oss-binaries.phusionpassenger.com
> PING juvia-helper.phusion.nl (109.107.35.58) 56(84) bytes of data.
> 64 bytes from juvia-helper.phusion.nl (109.107.35.58): icmp_seq=1 ttl=53 time=45.10 ms
> 64 bytes from juvia-helper.phusion.nl (109.107.35.58): icmp_seq=2 ttl=53 time=46.0 ms
> 64 bytes from juvia-helper.phusion.nl (109.107.35.58): icmp_seq=3 ttl=53 time=45.7 ms
> 64 bytes from juvia-helper.phusion.nl (109.107.35.58): icmp_seq=4 ttl=53 time=45.9 ms
> 64 bytes from juvia-helper.phusion.nl (109.107.35.58): icmp_seq=5 ttl=53 time=63.5 ms
> 64 bytes from juvia-helper.phusion.nl (109.107.35.58): icmp_seq=6 ttl=53 time=47.5 ms

> curl -I https://oss-binaries.phusionpassenger.com/apt/passenger/
> HTTP/2 200
> server: nginx/1.14.0 (Ubuntu)
> date: Sat, 25 Apr 2020 16:19:55 GMT
> content-type: text/html
> content-length: 38
> last-modified: Thu, 04 Feb 2016 17:03:53 GMT
> etag: "56b38479-26"
> strict-transport-security: max-age=31536000; includeSubdomains
> accept-ranges: bytes

but nothing, the repo is still down.

**Comment by FooBarWidget** [comment]
Thanks for reporting the issue. This has been fixed now.

**Comment by gr8bit** [comment]
Thanks a lot for the quick reply, my builds are working again :)

**Comment by soulraven** [comment]
hi, i confirm. The repo si ok now

**Comment by javims90** [comment]
Hi
![Screenshot_3](https://user-images.githubusercontent.com/63070104/81387023-49b30b00-9116-11ea-9eaf-8404b770790b.png)
,
It shows me the following error message (attached).
Can you help me.
Thank you

**Comment by FooBarWidget** [comment]
@javims90 It's oss-binaries, not oss.binaries

**Comment by MarkFChavez** [comment]
**EDIT**: Please ignore! Moved it in this [issue](https://github.com/phusion/passenger/issues/2279).

Not sure if this is the right place to report this but I'm encountering the same issue as discussed here before.

```
Err:20 https://oss-binaries.phusionpassenger.com/apt/passenger bionic/main amd64 Packages
  Connection failed [IP: 109.107.35.58 443]
Fetched 582 kB in 3min 4s (3156 B/s)
Reading package lists...
W: Failed to fetch https://oss-binaries.phusionpassenger.com/apt/passenger/dists/bionic/main/binary-amd64/Packages  Connection failed [IP: 109.107.35.58 443]
W: Some index files failed to download. They have been ignored, or old ones used instead.
```

I'm running this on a Ubuntu 18.04 server using DigitalOcean.

**Comment by wildanhuang** [comment]
It's happening again..
`Err:7 https://oss-binaries.phusionpassenger.com/apt/passenger bionic/main amd64 libapache2-mod-passenger amd64 1:6.0.4-1~bionic1                     
  Connection failed [IP: 109.107.35.58 443]
Fetched 1373 kB in 1min 1s (22.5 kB/s)
E: Failed to fetch https://oss-binaries.phusionpassenger.com/apt/passenger/pool/main/p/passenger/libapache2-mod-passenger_6.0.4-1~bionic1_amd64.deb  Connection failed [IP: 109.107.35.58 443]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
`

**Comment by CamJN** [comment]
it's fixed now. I'm going to lock this issue. if something happens in the future please open a new issue so it gets seen faster.
